TeamCode/src/main/java/org/firstinspires/ftc/teamcode/RobotConfig.kt

```diff
@@ -113,4 +113,10 @@ object RobotConfig {
         @JvmField
         var BOTSBURGH_FOLDER = File(AppUtil.ROOT_FOLDER, "/BotsBurgh")
     }
+
+    @Config
+    object StringLogging {
+        @JvmField
+        var BOTSBURGH_FOLDER = File(AppUtil.ROOT_FOLDER, "/BotsBurgh")
```

Comment:
`CsvLogging` and `StringLogging` both have the same `BOTSBURGH_FOLDER` variable. Can we combine them both into a single category? (Maybe named `Logging`?)